public/manifest.json

```diff
@@ -1,23 +1,6 @@
 {
   "short_name": "React App",
   "name": "Create React App Sample",
-  "icons": [
-    {
-      "src": "favicon.ico",
-      "sizes": "64x64 32x32 24x24 16x16",
-      "type": "image/x-icon"
-    },
-    {
-      "src": "logo192.png",
-      "type": "image/png",
-      "sizes": "192x192"
-    },
-    {
-      "src": "logo512.png",
-      "type": "image/png",
-      "sizes": "512x512"
-    }
-  ],
   "start_url": ".",
   "display": "standalone",
   "theme_color": "#000000",
```

Comment:
- [ ] As required all the boilerplate that got created by the CRA (*texts*, *images*, and *styles*) should be deleted, which means including `manifest.json` and `robots.txt` files. Please kindly delete them as well 🙃

Reply:
Deleted the said file as instructed